test/cli/lookup_test.py

```diff
@@ -381,6 +381,28 @@ def test_platform_lookup_ext(tmpdir):
     ]
 
 
+def test_platform_lookup_path(tmpdir):
```

Comment:
This test needs to be implemented for all other external files as well and possibly flagged as XFAIL or skipped for now.

And please do *not* try to avoid any redundancies between those tests. That will only complicate things down the road. If the lookup code is finally shared that can be easily done.

Reply:
sure.. but I want to get platform test working first before I start copy pasting.

Reply:
That is also my approach. The platform is the most dynamic but might still have issues. So the idea is to develop the shared code within platform but also address the issues within the existing code for all other cases so you have the expected results which should match (except for maybe some syntax) after switching over to the shared code. That's why I copy and paste all of them they are identical except for the options/files provided. So at the end you can just delete all the duplicates and make one of the tests parameterized.

Parameterizing them prematurely would start to assume on some (currently still unknown) final order/format and is counterproductive.